notebooks/tem_tutorial.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>image not showing up</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/compSPI/simSPI/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>